circuits/circom/utils/qv/stateLeafAndBallotTransformer.circom

```diff
@@ -27,8 +27,6 @@ template StateLeafAndBallotTransformer() {
     signal input slVoiceCreditBalance;
     // Signup timestamp.
     signal input slTimestamp;
-    // Timestamp indicating when the poll ends.
-    signal input pollEndTimestamp;
```

Comment:
given we removed this from this template, we should also remove `slTimestamp` from here and from `messageValidator`

Reply:
Will be removed in the next milestone!